src/simplewallet/simplewallet.cpp

```diff
@@ -1660,6 +1662,60 @@ bool simple_wallet::handle_command_line(const boost::program_options::variables_
 
   return true;
 }
+//----------------------------------------------------------------------------------------------------
+bool simple_wallet::prompt_for_new_node()
+{
+  bool connect = false;
+  std::string node_url;
+
+  std::string confirm = input_line(tr("Do you want to connect to a node?  (Y/Yes/N/No): "));
+
+  if (std::cin.eof())
+  {
+    LOG_ERROR("Unexpected std::cin.eof() - Exited simple_wallet::prompt_for_new_node()");
+  } else {
+	  connect = command_line::is_yes(confirm);
+  }
```

Comment:
If you're going to choose a style, it'd be nice to keep the same in a single if :)

Reply:
I've changed the code to two line indents, i must've missed this one. My bad. Gomenasai!

Reply:
There are several like that :)